src/api/java/com/minecolonies/api/util/CraftingUtils.java

```diff
@@ -55,23 +55,7 @@ public static int calculateMaxCraftingCount(@NotNull final ItemStack outputStack
      */
     public static int calculateMaxCraftingCount(@NotNull final int count, @NotNull final IRecipeStorage storage)
     {
-        //Calculate the initial crafting count from the request and the storage output.
-        int craftingCount = (int) Math.ceil(Math.max(count, ItemStackUtils.getSize(storage.getPrimaryOutput())) / (double) ItemStackUtils.getSize(storage.getPrimaryOutput()));
-
-        //Now check if we excede an ingredients max stack size.
-        for (final ItemStorage ingredientStorage : storage.getCleanedInput())
-        {
-            final ItemStack ingredient = ingredientStorage.getItemStack();
-            //Calculate the input count for the ingredient.
-            final int ingredientInputCount = ItemStackUtils.getSize(ingredient) * craftingCount;
-            //Check if we are above the max stacksize.
-            if (ingredientInputCount > ingredient.getMaxStackSize())
-            {
-                //Recalculate the crafting limit using the maxstacksize of the ingredient.
-                craftingCount = Math.max(ingredient.getMaxStackSize(), ItemStackUtils.getSize(storage.getPrimaryOutput())) / ItemStackUtils.getSize(storage.getPrimaryOutput());
-            }
-        }
-
-        return craftingCount;
+        //Calculate the crafting count from the request and the storage output.
+        return (int) Math.ceil(Math.max(count, ItemStackUtils.getSize(storage.getPrimaryOutput())) / (double) ItemStackUtils.getSize(storage.getPrimaryOutput()));
     }
```

Comment:
We used to have it to avoid subrequests < 64 items. We removed the limitations in other places but not here which caused some weird bugs. @OrionDevelopment